Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -596,13 +602,12 @@
                                     x:Name="ItemStatusGrid"
                                     Grid.Column="4"
                                     Width="{Binding ColumnsViewModel.StatusColumn.Length.Value, ElementName=PageRoot, Mode=OneWay}"
-                                    Padding="12,0,0,0"
                                     HorizontalAlignment="Stretch"
                                     VerticalAlignment="Center"
                                     Visibility="{Binding ColumnsViewModel.StatusColumn.Visibility, ElementName=PageRoot, Mode=OneWay}">
                                     <FontIcon
                                         x:Name="CloudDriveSyncStatusGlyph"
-                                        HorizontalAlignment="Left"
+                                        HorizontalAlignment="Center"
```

Comment:
There's still something strange with the alignment, doesn't look centered. Sidenote: we might reduce the default status column width, it's pretty large for showing an icon..

![image](https://user-images.githubusercontent.com/9673091/127752092-ab4e18b6-d98b-4b18-ade9-63c64cef692b.png)

Reply:
thanks, it should be properly lined up now
![image](https://user-images.githubusercontent.com/59544401/127752857-10b92cb7-25c1-4865-add6-b8fe1cad1b0c.png)